xinference/model/llm/llm_family.py

```diff
@@ -117,6 +117,30 @@ def cache(
             return cache_from_huggingface(llm_family, llm_spec, quantization)
 
 
+def cache_peft(peft_model_path: str) -> str:
```

Comment:
The name peft_model_path is misleading because it should actually be a Hugging Face ID. Would it be better to rename it to peft_model_id to avoid confusion for users?